tests/_fixtures/mesos/overlay-master.json

```diff
@@ -7,47 +7,47 @@
           "backend": {
             "vxlan": {
               "vni": 1024,
-              "vtep_ip": "198.15.0.1\/20",
+              "vtep_ip": "198.15.0.1/20",
```

Comment:
This works fine without the escaped characters, so I reverted it... I'm still not sure why I did that to begin with. Thanks for catching!